src/useIsShake.ts

```diff
@@ -1,18 +1,20 @@
-import React, { useState, useEffect } from 'react';
+import { useState, useEffect } from 'react';
 import { Accelerometer } from 'expo-sensors';
+import { Subscription } from 'expo-sensors/build/Pedometer';
+
+type Poll = number | null;
```

Comment:
Typing Polls array items.

Reply:
You can remove this comment. This is exactly what the code says